books/free-programming-books-bl.md

```diff
@@ -142,6 +143,14 @@
 * [ডেভসংকেত: বাংলা চিটশিটের ভান্ডার](https://devsonket.com) - Devsonket Team
 
 
+### MySQL
+
+* [Bangla MySQL Database Tutorials](https://www.youtube.com/playlist?list=PLTydW-y9HsbQ2ztoaLBJTd4wwjc_oqWx4) - Training with Live Project
+* [mysql bangla tutorial | Mysql Database | HSC,CSE, Anyone](https://www.youtube.com/playlist?list=PLbC4KRSNcMnqp4x6XstgFCVi6XVu37t99) - Learn Hunter
+* [MySql Database Bangla Tutorial Beginner to Advanced](https://www.youtube.com/playlist?list=PLH246IZCIBeA4h1R6fdgK06kj9lMb3joi) - Soft-All
+* [MySQL Database Bangla Tutorials (HSC student / Anyone)](https://www.youtube.com/playlist?list=PLgH5QX0i9K3qLcx9DvVDWmNJ7riPvxzCD) - Anisul Islam
```

Comment:
@anisul-Islam, thanks for your playlist

:warning: Add platform as seen in previous suggestions